# Connection cost to another cluster
uint32 target_cluster_id
float64 cost
float64 path_length
float64 yaw_change